.github/workflows/integration-tests.yaml

```diff
@@ -10,6 +10,7 @@ on:
       - ui/**
       - .nvmrc
       - Dockerfile
+      - flows/
```

Comment:
I think we should run these tests when we make changes to the `flows/` unless there is any opposition